PublicSDKAPITester/RCSystemInfoAPI.m

```diff
@@ -0,0 +1,54 @@
+//
+//  RCSystemInfoAPI.m
+//  PurchasesCoreSwift
+//
+//  Created by Joshua Liebowitz on 6/29/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+@import Foundation;
+@import Purchases;
+
+#import "RCSystemInfoAPI.h"
+
+@implementation RCSystemInfoAPI
```

Comment:
The great thing about the API tester is that it only sees public files since I have it setup to @import Purchases or @PurchasesCoreSwift
I build these files before I change the api, so if I'm understanding visibility correctly, what this project sees is what the public sees too.

Reply:
Will remove this.